dynamodb/dynamodb.tf-aws.w

```diff
@@ -56,7 +61,12 @@ pub class Table_tfaws impl dynamodb_types.ITable {
       attribute: props.attributes,
       hashKey: props.hashKey,
       rangeKey: props.rangeKey,
-      billingMode: "PAY_PER_REQUEST",
+      billingMode: "{this.billingMode}",
+      removalPolicy: "{this.removalPolicy}",
```

Comment:
Looking from the build error I don't think this is a valid field https://github.com/winglang/winglibs/actions/runs/9635881681/job/26605621599?pr=272

Reply:
Changed in c4d6624